recursive_art.py

```diff
@@ -80,9 +116,9 @@ def remap_interval(val,
         >>> remap_interval(5, 4, 6, 1, 2)
         1.5
     """
-    # TODO: implement this
-    pass
-
+    output_val =  input_interval_end - input_interval_start
```

Comment:
it would be useful to add more unit tests for this function, just to confirm that it works for all ranges.